index.html

```diff
@@ -16,59 +17,88 @@
 
 <body>
 
-  
-     
+
+
   <div class="container home">
-   
+    <header>
+      <nav>
+        <h1>Fun Bus </h1>
+        <a href="#home">Home</a>
+        <a href="#about">About Us</a>
+        <a href="#blog">Blog</a>
+        <a href="#contact">Contact</a>
+      </nav>
+      <img src="img/fun-bus.jpg" alt="Let's go on an adventure!">
+      <div>
+        <h2>Welcome To Fun Bus!</h2>
+        <p>Travelling Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign
+          adventure. Pretty
+          simple traveling fun WordPress wanderlust darn simple organized.</p>
+        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website
+          theme.</p>
+
+      </div>
+
+    </header>
+
     <section class="content-section">
       <div class="text-content">
         <h2>Let's Go!</h2>
-        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Pretty simple traveling fun WordPress wanderlust darn simple organized.</p> 
-        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p>
+        <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Pretty
+          simple traveling fun WordPress wanderlust darn simple organized.</p>
+        <p>Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website
+          theme.</p>
       </div>
       <div class="img-content">
         <img src="img/adventure.jpg" alt="Let's go on an adventure!">
       </div>
     </section>
     <section class="content-section inverse-content">
       <div class="img-content">
-          <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
+        <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
```

Comment:
Not sure if this piece of code came like this, but normally you only wrap anything in a div if you want to manipulate multiple elements inside the div. Since this <img> is by itself inside the div, you could just remove that div